e2e/tests/components/date-time-picker/date-time-picker.e2e-spec.ts

```diff
@@ -200,4 +200,13 @@ describe('Date Time Picker Tests', () => {
         }
 
     });
+
+    it('should not display horizontal line when select adjacent month', async () => {
+
+        let date = await page.getDate('31');
+        await date.click();
+
+        await page.nextDate.click();
+        expect(await imageCompare('date-picker-horizontal-line-removed')).toBe(0);
```

Comment:
The screenshot name should describe the use case, not the bug that you fixed. In this case, the Keppel theme has no horizontal line to remove so it doesn't make sense. In this case  `date-picker-adjacent-month-selection` might be appropriate.